db_test.go

```diff
@@ -450,7 +450,7 @@ func TestDB_Open_ReadOnly(t *testing.T) {
 	// Read from a read-only transaction.
 	if err := readOnlyDB.View(func(tx *bolt.Tx) error {
 		value := tx.Bucket([]byte("widgets")).Get([]byte("foo"))
-		if bytes.Compare(value, []byte("bar")) != 0 {
+		if !bytes.Equal(value, []byte("bar")) {
```

Comment:
already picked this up in e39821f3defb7f31a52ff48f5e1b01433dd713a3

Reply:
Great, I'll rebase.